partials/archive-event/introduction.php

```diff
@@ -0,0 +1,51 @@
+<?php
+$current_year_id = IGV_get_option('_igv_site_options', '_igv_current_fair_year');
+$current_year = (!empty($current_year_id)) ? get_term($current_year_id)->slug : false; 
+// if we have the term ID, get the slug, 
+// otherwise set it false for later conditionals
+
+$publish_program = IGV_get_option('_igv_page_options', '_igv_publish_program');
+
+$program_page_text = IGV_get_option('_igv_page_options', '_igv_program_page_text');
+$program_page_image_id = IGV_get_option('_igv_page_options', '_igv_program_page_image_id');
+?>
+
+<section id="program-intro" class="section">
+  <div class="container">
+    <div class="row">
+      <div class="col col-l col-l-12">
+        <h1>
+        <?php 
+          _e('[:en]Program&nbsp;[:es]Programa&nbsp;'); 
+          echo $current_year && $publish_program ? $current_year : ''; 
+          // add the current year to the heading if we have it, 
+          // and the exhibitors are published.
+        ?>
+        </h1>
+      </div>
+    </div>
+<?php 
+if (!empty($program_temp_text) || !empty($program_page_text) || !empty($program_page_image_id)) { 
```

Comment:
The `$program_temp_text` is never expressed no matter the status of if published or not. I feel like it would be better to just have a if for if published or not and then do the markup twice. But I'm not sure what the pre published view is supposed to look like